modal/image.py

```diff
@@ -868,6 +868,32 @@ async def _load(self: _Image, resolver: Resolver, existing_object_id: Optional[s
 
         return obj
 
+    async def build(self, app: "modal.app._App") -> "_Image":
+        """Eagerly build an image.
+
+        **Examples**
+
+        ```python
+        image = modal.Image.debian_slim().uv_pip_install("scipy", "numpy")
+
+        app = modal.App("build-image")
+        with app.run(), modal.enable_output():
```

Comment:
Normally you need to do `modal.enable_output()` before you do `app.run()`

```python
with modal.enable_output(), app.run():
    ...
```

I guess with this way you'd see the Image build logs but not any of the other "App status" output?